### DISCUSSION TITLE
Установка Тотум с подключением к выделенному серверу БД

### DISCUSSION THREAD

**Discussion by LostAlly** [totumonline/totum-ru-issues-and-discussions#345] [Как это сделать?] [answered]
Здравствуйте.
Не нашел ответа на этот вопрос ни тут ни в справке по установке. Ставить еще не пробовал, смотрел видео по автоматической установке.
Есть выделенный сервер БД, можно ли как-то при автоматической установке указать данные для подключения к внешнему серверу PostgreSQL?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Автоматический скрипт этого не умеет - ставьте пошагово: https://ru.docs.totum.online/ubuntu#manual

Если у вас там managed DB in cloud, то скорее-всего вам будут недоступны pgdump и psql и связанные с ними возможности консольнрй утилиты тотума
